CMakeLists.txt

```
@@ -97,6 +97,9 @@ endif()
 if(EXISTS "${OpenVINOGenAI_SOURCE_DIR}/tests/cpp")
```

```suggestion
if(EXISTS "${OpenVINOGenAI_SOURCE_DIR}/tests")
```
and add `CMakeLists.txt` to `tests`

---

Why is this check is needed in the first place? These folders are always here because we put add them

---

done